### PR TITLE
design updates

### PR DESCRIPTION
**Pull request by eballeste**
- [x] Overall fonts were matched with main site.
- [x] Topnav styles + responsive states.
- [x] OS Tabs redesigned. 
- [x] Anchor links now take into consideration fixed topnav. 
- [x] Footer was included.
- [x] Sidebar was adjusted to avoid overlapping the footer.

### CODE REVIEW

**Comment by jseldess** [comment]
Related email chain underway. Will post updates here soon. Thanks, @eballeste!

**Comment by jseldess** [comment]
@eballeste, just sent an email with feedback. Copying here as well. Thanks, Jesse

Hi Enrique, Dwight, Andrew,

Spencer, Jessica, and I have synced up. The topnav, OS toggles, and footer look great. There are just a few other changes we'd like your help with:

- **Headers:** We know it's a little inconsistent with the header style on the wordpress site, but in the docs, we'd like the headers to remain sans serif and sized as before. 
- **Body text:** It's great that you matched the font, but I'd like to keep the sizing consistent. In your version, I think bulleted text, numbers, and text in callouts are smaller. Can you just make them consistent with the rest of the body text?
- **Whitespace at top:** I'd prefer that we have more whitespace between the topnav and the top of the sidebar and page header. Can you please revert to something closer to what you see [here](http://cockroach-draft-docs.s3-website-us-east-1.amazonaws.com/frequently-asked-questions.html)?
- **Footer:** When a page is empty, the footer overlaps the sidebar. Here's an [example](http://cockroachlabs.divisionof.com/docs/configure-replication-zones.html). Can you change it so the footer always remains below either the sidebar or content on the page, whichever is longer?
- **Left/right margin at normal width:** When I compare your example site to the live site, it looks like your topnav is a little too narrow, whereas the body is a little to wide. Can you please see if we can make it visually seamless, in terms of left/right margin, when moving from any page on the wordpress site to any page on the docs site? 
- **Left/right margin at low width:** When you shrink the page to less than 758px wide, there's no longer any margin around the content. Instead, can you replicate what we do at cockroachlabs.com (we keep a small margin). 

Jessica's got another thread going with you, or soon will, about changing the topnav to have only one state (thin) and redesigning the footer. One other topic we'll need to tackle is:

- **Auto-selecting the correct OS tab on the Installing CockroachDB page:** When a user clicks the Install button, or just lands on the install cockroachdb page, we'd like to recognize the user's OS and automatically select the matching tab. I think we may have discussed this in passing at one point. Is this something you all could work in?

Thanks again!

**Comment by eballeste** [comment]
- [x] Header styles reverted to original look with sans-serif font
- [x] Body text font-size normalized
- [x] Top whitespace adjusted
- [x] Footer overlap fixed
- [x] Margins adjusted for seamless left and right alignment between header and content
- [x] Mobile margins fixed
- [x] OS-Tab default state now takes into consideration users OS

**Comment by spencerkimball** [comment]
Where's the staging?

**Comment by eballeste** [comment]
Sorry, here you go:
[staging](http://cockroachlabs.divisionof.com/docs/)

File: _includes/topnav.html [hunk 1]
```diff
@@ -1,106 +1,37 @@
...
-            <ul>
-                <li><a href="{{ "/frequently-asked-questions.html" | prepend:site.baseurl }}">FAQ</a></li>
-                <li><a href="http://www.cockroachlabs.com/docs/">Docs</a></li>
-                <li><a href="http://www.cockroachlabs.com/community/">Community</a></li>
-                <li><a href="http://www.cockroachlabs.com/about/">About</a></li>
-                <li><a href="http://www.cockroachlabs.com/blog/">Blog</a></li>
-                <li><a href="{{ "/install-cockroachdb.html" | prepend:site.baseurl }}" class="btn">Install</a></li>
-            </ul>
-        </div>
-<!--    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
```
Comment (jseldess) [inline _includes/topnav.html]: Please leave this commented-out code in place. There's some conditional logic that plugs into some of the theme's conditional logic capabilities, which we don't need now but could come in handy later.

**Comment by jseldess** [comment]
Hey @eballeste, 

Thanks for making these changes. Overall, looks great! Other than putting the commented-out code back in (mentioned above), I have just a few more requests:

- **Top margin for sidebar:** When the topnav is in thin state, let's make sure the sidebar still has a margin-top of 40px. 
- **Footer:** It looks like there are significantly fewer links in **Connect** section of the live footer at cockroachlabs.com. I'm not sure which version is correct. Can you look into this? @jess-edwards might need to chime in. Also, when you hover over links in the live footer, you get an underline. That's not happening in this branch. Can you please fix that?

Also, it looks like you need to do another update from master to resolve conflicts. Once you've done that and handled the things above, let me know and I'll merge. Then, once you all have made the other changes to the wordpress site (only thin-state topnav, thinner footer, etc.), you can make those changes to the docs portion as well and submit a pull request. 

Thanks again!
Jesse

**Comment by eballeste** [comment]
- [x] Included commented code with some revisions and markup cleanup due to unclosed div tags/extra liquid unclosed "for loops", also replaced regular html comment tags with liquid comment tags
- [x] Top margin for sidebar increased when upper nav is in thin state
- [x] Footer links were commented out to match those found on live site

**Comment by eballeste** [comment]
Scratch that, I'm updating the code to take into consideration that the sidebar nav's content will keep on changing in the future as you keep on building the docs and that this value should affect the min-height value of the main content div. This will help avoid the overlapping of the sidebar with the footer. (currently using a static value for the min-height of the content div)

**Comment by eballeste** [comment]
The sidebar overlapping the footer should not be an issue anymore. You can see the new behavior by clicking on the longer submenu item "learn how it works": [here](http://cockroachlabs.divisionof.com/docs/explore-the-admin-ui.html)

**Comment by jseldess** [comment]
Thanks, @eballeste. Looks great. Can you update from master again? Once there are no conflicts, I can merge.

**Comment by eballeste** [comment]
Hey @jseldess, our branch has been updated with your last commits.

**Comment by jseldess** [comment]
Awesome. Thanks, @eballeste. I just merged.

**Comment by jseldess** [comment]
@eballeste: after merging and running the new site locally, I noticed that the footer is wonky. I've sync to our amazon site so you can see what I mean: http://cockroach-draft-docs.s3-website-us-east-1.amazonaws.com/. 

Can you please take a look and fix asap?

**Comment by jseldess** [comment]
Nevermind, @eballeste. We're including the footer in the page layout and the default layout, so I'll just remove it from the page layout.

**Comment by eballeste** [comment]
Hi! I know exactly where the problem is... it happened on this last final conflict fix and I apologize for not having noticed it but in _layouts/page.html line 41 should not be there.

`{% include footer.html %}`

**Comment by eballeste** [comment]
Ok, looks like you got it.

**Comment by jseldess** [comment]
Yep. Saw the same thing. Thanks, Enrique.

**Pull request merged**